main.py

```diff
@@ -58,31 +61,48 @@ def shutdown(self):
     def trace(self):
         """Trace a path."""
         entries = request.get_json()
+        entries = convert_entries(entries)
```

Comment:
Gretel, let's get rid of any of the translations https://github.com/kytos-ng/sdntrace_cp/pull/47#discussion_r995759094, we don't need them, keeping them since we'll nuke `GenericFlow` will only make the code more difficult to maintain.